app/src/main/assets/extensions/ads/manifest.template.json

```diff
@@ -19,9 +19,20 @@
       "run_at": "document_end"
     }
   ],
+  "background": {
+    "scripts": ["adsBackground.js"]
+  },
   "permissions": [
     "geckoViewAddons",
     "nativeMessaging",
-    "nativeMessagingFromContent"
+    "nativeMessagingFromContent",
```

Comment:
Permissions are also reused from the cookies https://github.com/mozilla-mobile/fenix/blob/07748f69b75144e8e247b742f66a756f999cb299/app/src/main/assets/extensions/cookies/manifest.template.json.